src/main/java/org/tron/core/Constant.java

```diff
@@ -51,8 +51,11 @@
   public static final long MAX_CPU_TIME_OF_ONE_TX = 50000; // 50 ms = 50000 us
   public static final long STORAGE_LIMIT_IN_ONE_TX_OF_SMART_CONTRACT = 32 * 1024 * 1024L; // 32MB
   public static final long SUN_PER_ENERGY = 100; // 1 us = 100 DROP = 100 * 10^-6 TRX
-  public static final long MAX_ENERGY_IN_TX = 3000000; // ref: 1 us = 1 energy
+  public static final long ENERGY_LIMIT_IN_CONSTANT_TX = 3000000; // ref: 1 us = 1 energy
   public static final long MAX_RESULT_SIZE_IN_TX = 64; // max 8 * 8 items in result
+  public static final long PB_DEFAULT_ENERGY_LIMIT = 0;
+  public static final long CREATOR_DEFAULT_ENERGY_LIMIT = 1000 * 10_000;

```

Comment:
![Bug](https://sonarsource.github.io/sonar-github/bug.png 'Bug') Bug: Cast one of the operands of this multiplication operation to a "long". ([squid:S2184](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AS2184))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1670&issues=AWbSdHXNSgN4uCclEWnk&open=AWbSdHXNSgN4uCclEWnk)

[ISSUE_KEY]: AWbSdHXNSgN4uCclEWnk